pandas/io/excel.py

```diff
@@ -191,12 +193,12 @@ def get_writer(engine_name):
 
 
 @Appender(_read_excel_doc)
-def read_excel(io, sheet_name=0, header=0, skiprows=None, skip_footer=0,
-               index_col=None, names=None, parse_cols=None, parse_dates=False,
-               date_parser=None, na_values=None, thousands=None,
-               convert_float=True, converters=None, dtype=None,
-               true_values=None, false_values=None, engine=None,
-               squeeze=False, **kwds):
+def read_excel(io, sheet_name=0, header=0, skiprows=None, nrows=None,
+               skip_footer=0, index_col=None, names=None, parse_cols=None,
```

Comment:
I rearranged the kwargs and also updated docstrings to match parameter order (one of the things that always bugs me). Should I also update all internal function kwargs in ```excel.py``` to match the external API?

I'm adding the following note to the Other Enhancements section (just wanted to make sure it was the right spot...):
```- Rearranged the order of keyword arguments in :func:`read_excel()` to align with :func:`read_csv()` ```

Thanks!